src/callbacks_step/analysis_surface_integral_2d.jl

```diff
@@ -13,24 +13,28 @@
                                                           boundary_symbol_or_boundary_symbols,
                                                           variable)
 
-    This struct is used to compute the surface integral of a quantity of interest `variable` alongside
-    the boundary/boundaries associated with particular name(s) given in `boundary_symbol`
-    or `boundary_symbols`.
-    For instance, this can be used to compute the lift [`LiftCoefficientPressure`](@ref) or
-    drag coefficient [`DragCoefficientPressure`](@ref) of e.g. an airfoil with the boundary
-    name `:Airfoil` in 2D.
+This struct is used to compute the surface integral of a quantity of interest `variable` alongside
+the boundary/boundaries associated with particular name(s) given in `boundary_symbol`
+or `boundary_symbols`.
+For instance, this can be used to compute the lift [`LiftCoefficientPressure`](@ref) or
+drag coefficient [`DragCoefficientPressure`](@ref) of e.g. an airfoil with the boundary
+name `:Airfoil` in 2D.
```

Comment:
Remove indentation to have this correctly rendered, see 

https://trixi-framework.github.io/Trixi.jl/dev/reference-trixi/#Trixi.AnalysisSurfaceIntegral